client/scripts/app.ts

```diff
@@ -228,12 +228,7 @@ export async function selectNode(n?: NodeInfo, deferred = false): Promise<boolea
   if (app.chain && n === app.chain.meta) {
     return;
   }
-  if ((Object.values(ChainNetwork) as any).indexOf(n.chain.network) === -1
```

Comment:
Why is this removed?

Reply:
With this line of code in, it's impossible to load a dynamically created Substrate chain, as it mandates that all chains must be explicitly numerated in the `ChainNetwork` list, aside from tokens. This check for undefined chains seems redundant with the check on line 305, so it should be fine to delete.